docs/jetpack-compose.md

```diff
@@ -1,7 +1,37 @@
 # Jetpack Compose
 
-Although Jetpack Compose has not hit 1.0 yet, ensuring that Mavericks will be compatible with it is extremely important. Alongside Compose there is value in using Mavericks as a container for business logic that will pair well with a tree of composable UI nodes.
+Mavericks is still useful in a Jetpack Compose world. With Jetpack Compose, you can use Compose for all of your UI but Mavericks for your business logic, data fetching, interface to your dependency injection object graph, etc.
 
-We haven't settled on a final API yet but have already begin to explore what one might look like. [Here](https://gist.github.com/gpeal/5bed618a843d4d71bb1518fa8317c5df) are some snippets of working code that enables you to use Mavericks with Compose today. It is likely that the final API will look extremely similar to this.
+To use Mavericks with Jetpack Compose, add the `com.airbnb.android:mavericks-compose` artifact (TBD on versioning).
 
-When Compose and the associated tooling (Android Studio, Android Gradle Plugin, etc) are closer to being finalized, we will release an official Compose artifact.
+To get started, create a MavericksViewModel just like you always have. Now, with the `mavericks-compose` artifact, you can get or create a ViewModel in a composable like this:
+
+```kotlin
+@Composable
+fun MyComponent() {
+    val viewModel: CounterViewModel = mavericksViewModel()
+    val count by viewModel.collectState(CounterState::count)
+    Button(onClick = activityScopedViewModel::incrementCount) {
+        Text("Count: $count")
+    }
+}
+```
+
+If you call `collectAsState()` with no parameters, it will return a Compose state property with the whole ViewModel state class and will update any time the state changes.
+
+The recommended approach is to subscribe individually to a small set of State properties and to break down large Composables into smaller ones. This allows Compose to more efficiently recompose your UI when state changes.
```

Comment:
Do we provide utils to easily subscribe to nested properties for efficiency? For example `viewModel.collectState(CounterState::listing).map { it.host }`

Reply:
looks like this uses a compose State, so I assume that has mapping. maybe worth calling out as an example

Reply:
I just added an overload to allow you to do `collectState { it.listing.host }` which would allow for this. You could also make host a derived prop.